mdr/mdr.py

```diff
@@ -73,7 +90,31 @@ def transform(self, features):
             Constructed features from the provided feature matrix
 
         """
-        return 0
+        new_feature = np.zeros(shape=(features.shape[0],1), dtype=np.int)
```

Comment:
I try committing again and I think all of them are there! The last one does not mark as an X but I made sure I cut down on that function too, it is just that line that was not changed.